CONTRIBUTING.rst

```diff
@@ -182,14 +181,13 @@ Do these things for each feature or bug
   - Click 'Create Pull Request`
 
 Then, a cloud-init committer will review your changes and
-follow up in the pull request.  Look at the `Code Review Process`_ doc
+follow up in the pull request.  Look at the :ref:`Code Review Process<code-review-process>` doc
```

Comment:
`tox -re doc` fail in CI and locally. I think because your anchor name is hyphenated instead of underscore delmited.